magento/magento2
# magento/magento2#35881: Final Price for grouped products does not include tax in the Recently Viewed Widget

### Description (*)

1. In Magento\Catalog\Ui\DataProvider\Product\Listing\Collector\Price::collect Added a product type check. If it's a grouped, I choose child product with the final minimum price
2. For Grouped product in file GroupedProduct/view/frontend/ui_component/widget_recently_viewed.xml need add empty special_price template. If you don't edit it, the price block is duplicated in the widget.
3. In  Catalog/view/frontend/templates/product/view/opengraph/general.phtml meta tag <meta property="product:price:amount"  add product type check. If it's a grouped, I choose child product with the final minimum price
 
### Related Pull Requests
https://github.com/magento-gl/magento2-page-builder/pull/28


### Fixed Issues (if relevant)

1. Fixes magento/magento2#35881

### Manual testing scenarios (*)

1.  made sure that tax classes are set for taxable goods
2. I made sure that I see the prices both with and without tax
5. Visit sample grouped product
6. Go back to the home page to view the Recently Viewed Block
7. Go to the Grouped product page. Find the tag (<meta property="product:price:amount" ) in the browser code inspector - and check the price

### Questions or comments


### Contribution checklist (*)
